services/release/release_test.go

```diff
@@ -131,7 +131,7 @@ func TestRelease_Update(t *testing.T) {
 	releaseCreatedUnix := release.CreatedUnix
 	time.Sleep(2 * time.Second) // sleep 2 seconds to ensure a different timestamp
 	release.Note = "Changed note"
-	assert.NoError(t, UpdateReleaseOrCreatReleaseFromTag(user, gitRepo, release, nil, false))
+	assert.NoError(t, UpdateReleaseOrCreatReleaseFromTag(user, gitRepo, release, nil, nil, nil, false))
```

Comment:
Can we add tests for each of these nil values so that this doesn't break in the future? If it's essential to merge these quickly I can open a new PR with tests for these.

Reply:
@kdumontnu best practice is to create a pull against lunny's feature branch :)

Reply:
Tests added.